nest-tax-backend/src/card-payment-reporting/card-payment-reporting.service.ts

```diff
@@ -125,6 +125,7 @@ export class CardPaymentReportingService {
     const rows = parse(csvContent, {
       delimiter: ';',
       fromLine: 2, // Skip first line
+      relax_column_count_less: true,
```

Comment:
This `snake_case` is incorrect. It won't even cause an error, it will be just be ignored. The supported syntax in our case is cammelCase. Please change to `relaxColumnCountLess`.

Reply:
After further review this seems to be untrue, however, please still use the the `camelCase` for consistency reasons, as the line right above with `fromLine` also uses `camelCase`.